tests/react/vanilla-utils/atomWithStorage.test.tsx

```diff
@@ -445,6 +445,62 @@ describe('atomWithStorage (with browser storage)', () => {
 
     expect(store.get(dummyAtom)).toBe(1)
   })
+
+  it("should recompute dependents' state after onMount (#2098)", async () => {
+    const store = createStore()
+    const mockNativeStorage = Object.create(window.Storage.prototype)
+    let currentValue: string | null = 'true'
+    mockNativeStorage.setItem = vi.fn(
+      (_key: string, value: string) => (currentValue = value)
+    )
+    mockNativeStorage.getItem = vi.fn(() => currentValue)
+    mockNativeStorage.removeItem = vi.fn(() => (currentValue = null))
```

Comment:
Can it be something like this?
```suggestion
    let currentValue: string | null = 'true'
    const mockNativeStorage = {
      setItem: vi.fn(
      (_key: string, value: string) => (currentValue = value)
    ),
      getItem: vi.fn(() => currentValue),
      removeItem: vi.fn(() => (currentValue = null)),
    }
```

Reply:
updated, it's copied from https://github.com/pmndrs/jotai/blob/2ec0bb96533f827b2b6f38456bbeab9cc542eddc/tests/react/vanilla-utils/atomWithStorage.test.tsx#L380-L383, should I update it too?